editor/src/themes/index.scss

```
@@ -63,6 +63,7 @@
 @import 'gender-question/gender-question.scss';
 @import 'buttons/button.scss';
 @import 'buttons/btn-group.scss';
+@import 'buttons/btn-groupWithSVG.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)